Marlin/src/pins/pins_lcd.h

```diff
@@ -49,6 +49,8 @@
     #include "lcd/RRD_FG_SC.h"
   #elif IS_RRD_SC
     #include "lcd/RRD_SC.h"
+  #elif ANY(FYSETC_MINI_12864_X_X,FYSETC_MINI_12864_1_2,FYSETC_MINI_12864_2_0,FYSETC_MINI_12864_2_1,FYSETC_GENERIC_12864_1_1) // Also covers MKS_MINI_12864_V3 and BTT_MINI_12864_V1
```

Comment:
New LCDs that might be `IS_RRD_FG_SC` and/or `IS_RRD_SC` should be added above `IS_RRD_FG_SC`.

Reply:
`ENABLED(FYSETC_MINI_12864)` is fine here because it is provided for us by `Conditionals_LCD.h`.

Reply:
ok.